src/main/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvert.java

```diff
@@ -93,34 +72,14 @@ public static String getAddressFromJson(String jsonString) {
 		return result.get("address").toString();
 	}
 
-	/**
-	 * Converts a JSON expression to an Apartment object.
-	 *
-	 * @param jsonPath<i>Path</i> the JSON expression to convert into Apartment
-	 *                            object
-	 * @return <i>Apartment</i> the Apartment generated
-	 * @throws IOException if the file can't be red
-	 */
-	public static Apartment jsonToApartment(Path jsonPath) throws IOException {
-		String jsonString = Files.readString(jsonPath);
-		Jsonb jsonb = JsonbBuilder.create();
-		LOGGER.info("Create Json builder");
-
-		Apartment.Builder apartBuild = jsonb.fromJson(jsonString, Apartment.Builder.class);
-		LOGGER.info("Apartment created from JSON");
-
-		return apartBuild.build();
-	}
-
 	/**
 	 * Converts a JSON expression to a list of Apartments.
 	 *
 	 * @return <i>List</i> the list of Apartments created
-	 * @throws IOException        if the file doesn't exists
-	 * @throws URISyntaxException
+	 * @throws IOException if the file doesn't exists
 	 */
 	public static List<Apartment> jsonToApartments() throws IOException {
```

Comment:
`getDefaultApartments()` me semble plus parlant.

Reply:
Fix in commit ec00a78df49269994853daaed9f9ff607989f3ca